packages/schematics/src/collection/application/index.ts

```diff
@@ -117,9 +119,27 @@ Nx is designed to help you create and build enterprise grade Angular application
     const content = options.routing
       ? `${baseContent}\n<router-outlet></router-outlet>`
       : baseContent;
-    host.overwrite(
-      `${options.appProjectRoot}/src/app/app.component.html`,
-      content
+
+    if (!options.inlineTemplate) {
+      return host.overwrite(
+        `${options.appProjectRoot}/src/app/app.component.html`,
+        content
+      );
+    }
+
+    const modulePath = `${options.appProjectRoot}/src/app/app.component.ts`;
+    const templateNodeValue = getDecoratorPropertyValueNode(
+      host,
+      modulePath,
+      'Component',
+      'template',
+      '@angular/core'
+    );
+    replaceNodeValue(
+      host,
+      modulePath,
+      templateNodeValue,
+      `\`\n${baseContent}\n\`,\n`
```

Comment:
why do we need the comma? (`,\n`) Is it because the comma is part of the same node?

Reply:
Yes, the comma get stripped during the transformation, so i needed to add it back.